rqt_gui_py/src/rqt_gui_py/ros_py_plugin_provider.py

```diff
@@ -65,8 +68,11 @@ def _init_node(self):
         if not self._node_initialized:
             name = 'rqt_gui_py_node_%d' % os.getpid()
             qDebug('RosPyPluginProvider._init_node() initialize ROS node "%s"' % name)
+
             rclpy.init()
             self._node = rclpy.create_node(name)
+            self.spinner = RclpySpinner(self._node)
```

Comment:
This line should probably go into the constructor to ensure `spinner` is always set. Otherwise `shutdown` can't assume it exists.

Reply:
This was addressed in PR #163